### DISCUSSION TITLE
../unsandboxed-extensions redirects

### DISCUSSION THREAD

**Discussion by yuri-kiss** [TurboWarp/.github#2] [General]
i had no idea where to put this due to there not being a discussions tab on the turbowarp.org page,
I was wondering why experiments.turbowarp.org/unsandboxed-extensions/ redirects to turbowarp.org

**Comment by GarboMuffin** [comment]
Because unsandboxed extensions are no longer an experiment. turbowarp.org fully supports unsandboxed extensions.

**Comment by yuri-kiss** [comment]
yeah but some extensions are unable to get into the gallery so thats what was used, if you can cant you add a url param or something to enable ALL extensions even ones not on the gallery to be ran unsandboxed?
________________________________

**Reply by GarboMuffin** [reply]
The unsandboxed extensions experiment had the exact same limitations.

**Reply by GarboMuffin** [reply]
There are inherent unfixable security risks in letting people run arbitrary untrusted JS. If someone's account gets phished through that, suddenly it becomes our fault.

**Reply by GarboMuffin** [reply]
We let you run unsandboxed extensions from local files now. That should be good enough for 90% of uses.

**Reply by yuri-kiss** [reply]
well that sucks, since you cant do that is there at least a way to fork turbowarp and disable the sandboxing for my own purposes?

**Reply by yuri-kiss** [reply]
as idk how to host a NodeJS site on github
